Jenkinsfile

```diff
@@ -226,13 +216,23 @@ def job = {
                                 sh cmd
                             }
                             step([$class: 'hudson.plugins.findbugs.FindBugsPublisher', pattern: '**/*bugsXml.xml'])
+
+                            if (!config.isPrJob) {
+                                def git_tag = "v${config.docker_tag}-ksqldb"
```

Comment:
seems odd to use the docker_tag here instead of the artifact version, even though they are the same thing.

Reply:
Don't feel strongly one way or the other. I've updated it to be the artifact version.